android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -676,6 +676,18 @@ public void scrollToBottom()
 		}
 	}
 
+	public void scrollToTop()
+	{
+		View view = getNativeView();
```

Comment:
Once we update this PR with what's on master, we'll need to change this line of code...

```java
View view = getNativeView();
```

...to...

```java
View view = this.scrollView;
```

This is because we've added `RefreshControl` support to scroll views as of 6.3.0 and the `getNativeView()` method will return a `SwipeRefreshLayout` object instead.